webpack.config.prod.js

```diff
@@ -29,17 +30,13 @@ module.exports = {
       {
         test: /\.(jpe?g|png|gif)$/i,
         use: {
-          loader: 'url-loader?limit=10000!img?progressive=true&-minimize',
-          options: {
-            name: '[name].[ext]',
-            outputPath: 'img/'
-          }
+          loader: 'url-loader?limit=10000!img?progressive=true&-minimize&outputPath=./img/',
```

Comment:
I tested reducing this limit but there are a lot of images that are inlined by imported `.usa` styles that we aren't impacted by this rule. We could potentially reduce the size of our CSS by finding a way to manage the inlining of these images.

Reply:
I also don't know what `!img` does.